### PR TITLE
Fix GLTF skinning and morphing

### PR DESCRIPTION
**Pull request by NolaDonato**
1. Transpose inverse bind matrix in assimp
2. Do not use glm::decompose in setModelMatrix, it produces negative scaling
3. Increase maximum number of blend weights to 200

SXR DCO signed off by: Nola Donato nola.donato@samsung.com

### CODE REVIEW

File: SXR/SDK/sxrsdk/src/main/jni/objects/components/transform.cpp [hunk 1]
```diff
@@ -121,14 +121,18 @@ void Transform::setModelMatrix(glm::mat4 matrix) {
...
     glm::quat rotation;
     glm::vec3 translation;
-    glm::vec3 skew;
-    glm::vec4 perspective;
-    glm::decompose(matrix, scale, rotation, translation, skew, perspective);
 
     mutex_.lock();
-    position_ = translation;
-    scale_ = scale;
-    rotation_ = glm::conjugate(rotation);
```
Comment (liaxim) [inline SXR/SDK/sxrsdk/src/main/jni/objects/components/transform.cpp]: Keep in mind the conjugate is needed to fix issue https://github.com/Samsung/GearVRf/pull/1969. Unless you have another fix quat_cast is not going to work.

**Comment by NolaDonato** [comment]
I have tested with the application in issue #1969 and my version works. The glm::conjugate call is not needed.

**Comment by NolaDonato** [comment]
All of the asset tests pass now except for jassimpCubePointDiffusePoint. The latest version of assimp positions the light in negative Z (behind the object) instead of positive Z (in front of the object). I am looking into this regression.

**Comment by NolaDonato** [comment]
I figured out the point light regression and updated the test. All the asset tests pass with this PR now.

**Comment by NolaDonato** [comment]
I have fixed the JNI crash in sxr-morph

**Comment by NolaDonato** [comment]
I merged in the change to travis.yml

**Comment by liaxim** [comment]
Builds fine; for travis I just removed the licences; Travis supposedly automatically accepts all license prompts.

**Comment by liaxim** [comment]
sxr-morph not centered; please look into that.

Screen shot taken while having looking down about 45 degrees

morph

**Pull request merged**